scripts/apps/authoring/views/send-item.html

```diff
@@ -100,6 +104,27 @@
         </ul>
     </div>
 
+    <div class="slide-pane__footer" sd-loading="loading" ng-show="currentUserAction === vm.userActions.externalsource_to">
+        <ul class="btn-list">
+            <li class="full-width" ng-if="showSendButtonAndDestination() && !canSendItem()">
+                <button id="externalsource-open-btn" class="btn btn--light-gray"
+                        ng-click="send(true)"
+                        ng-disabled="disableFetchAndOpenButton()"
+                        translate>
+                    get from external source and open
+                </button>
+            </li>
+            <li class="full-width" ng-if="showSendButtonAndDestination()">
+                <button id="externalsource-btn" class="btn btn--primary"
+                        ng-click="send()"
+                        ng-disabled="disableSendButton()"
```

Comment:
Like above, please double check if `disableSendButton` is relevant if the story is retrieved from external source.